File: codechef/week_1/mini_piza.py
def gcd(a, b):
    while b:
        a, b = b, a % b
    return a


t = int(input())

while t:
    n, k = [int(i) for i in input().split(" ")]
    print(int(n / gcd(n, k)))
    t -= 1

File: codechef/week_1/mini_piza.go
package main

import "fmt"

func gcd(a int, b int) int {
	for b != 0 {
		a, b = b, a%b
	}
	return a
}

func main() {
	var t int
	fmt.Scanln(&t)
	for i := 0; i < t; i++ {
		var n, k int
		fmt.Scan(&n, &k)
		fmt.Println(n / gcd(n, k))
	}
}
